### ISSUE TITLE
Inquiries about using two-dimensional ultrasound image as dataset

### ISSUE THREAD

**Issue by shakerxuek** [microsoft/Recursive-Cascaded-Networks#52]
Dear developer，

Thank you for your time to read my inquiries. I managed to use two-dimensional ultrasound image as dataset to get weights. And I  encountered several problems.

The proposed method to handle input is introduceing h5 file and json file. When I tried to build dataset by creating h5 file with jpg ultrasound image，it causes various errors. I understand that is because the proper input format is dicom. But what should I do to create h5 file as dataset with two-dimensional image? How did you create h5 file as dataset? 

Thank you in advance, I am looking forward to receive your response.

**Comment by zsyzzsoft** [comment]
You can first inspect the format (keys, shapes, data types) of our provided h5 files and then create your dataset following our format. However, you need to change a lot of things other than the dataset to work with 2D images.

**Comment by shakerxuek** [comment]
Thanks for your reply, I tried to stacked my 2D ultrasound image and convert them into 3D DICOM file. I have create h5 file as training set following you format, and I noticed that the h5 files used for testing purpose (like lits_val.h5) have 3 members in each group while the training set has 1. What is the purpose of this structure?

**Comment by zsyzzsoft** [comment]
The validation set also contains segmentation mask (to evaluate the Dice score) and landmarks (to evaluate the landmark distance) for testing purpose.
